app/views/posts/home.html.erb

```diff
@@ -17,10 +18,10 @@
 		<div >
 			<ul class="posts-list">
 				<% @posts.reverse.each do |post| %>
-				<li>
+				<li class="posts-list-item">
 					<div>
 
-						<p><%= post.content %></p>
+						<h3><p><%= post.content %></p></h3>
 						<p><%= post.user.email %></p>
```

Comment:
**Suggestion:** It's a good practice to sanitize user-generated content to prevent Cross-Site Scripting (XSS) attacks. Use the h (short for html_escape) method to escape any potentially harmful HTML before rendering it.
```suggestion
      <h3><p><%= h post.content %></p></h3>
      <p><%= h post.user.email %></p>
```